src/main/java/com/google/api/codegen/csharp/CSharpGapicContext.java

```diff
@@ -633,6 +662,19 @@ private String getTypeName(ProtoElement elem) {
             }
           }
         });
+    if (isPageStreaming) {
+      String[] pageToken = {
+          "/// <param name=\"pageToken\">The token returned from the previous request.",
+          "/// <c>Null</c> or an empty string retrieves the first page.</param>",
+      };
+      String[] pageSize = {
+          "/// <param name=\"pageSize\">The size of page to request, the response will not be larger",
```

Comment:
Nit - I'd either use two sentences, or at least change the comma to a semi-colon.